### PR TITLE
Adds bounced email and suppression retrieval, and suppression activation...

### PR DESCRIPTION
**Pull request by jnormore**
- Adds bounced email count and data retrieval methods
- Adds suppression count and data retrieval methods
- Adds method to activate a suppressed email

@jduff Can you please review, thanks!

### CODE REVIEW

File: lib/dynect_email.rb [hunk 1]
```diff
@@ -4,11 +4,14 @@ module DynectEmail
   class Error < StandardError; end
 
   include HTTParty
+  # debug_output $stderr
```
Comment (jduff) [inline lib/dynect_email.rb]: can we remove this?

File: lib/dynect_email.rb [hunk 2]
```diff
@@ -43,4 +72,11 @@ def self.post_data(url, options={}, apikey=nil)
 
     handle_response(result)
   end
+
+  def self.get_data(url, options={}, apikey=nil)
```
Comment (jduff) [inline lib/dynect_email.rb]: This and the method above are basically the same. we can unify them.

``` ruby
def self.request(http_method, url, options={}, apikey=nil)
  options.merge!(:apikey => apikey || DynectEmail.api_key)
  result = case http_method
  when :post
    post(url, :query => options)
  when :get
    get(url, :query => options)
  else
    raise "Invalid HTTP method"
  end
end
```

Or, something like that.

**Comment by jnormore** [comment]
@jduff Updates for common request method comment removal, thanks

**Comment by jduff** [comment]
:shipit:

**Pull request merged**